src/lib/constants.js

```diff
@@ -48,6 +48,7 @@ export const STACKS_API_WS_URL = localNode
 export const STACKS_API_V2_INFO = `${STACKS_API_URL}/v2/info`;
 export const STACKS_API_ACCOUNTS_URL = `${STACKS_API_URL}/v2/accounts`;
 export const STACKS_API_FEE_URL = `${STACKS_API_URL}/v2/fees/transfer`;
+export const TESTNET_FAUCET_URL = 'https://explorer.stacks.co/sandbox/faucet';
```

Comment:
I think this URL needs to include `?chain=testnet` or it may cause an error.

```suggestion
export const TESTNET_FAUCET_URL = 'https://explorer.stacks.co/sandbox/faucet?chain=testnet';
```

I'm logged into the sandbox and when accessing this link after interacting with mainnet, and when requesting STX from the faucet I receive an error. I did notice that if I pull up my STX profile it shows mainnet information.

If I append the code above then my request for testnet STX succeeds and the profile shows my testnet balance.

Reply:
addressed in [
1d3f2ed](https://github.com/citycoins/citycoin-ui/pull/115/commits/1d3f2ed739db7c07e52ca819c44c7d176a2affd0)